GameData/RealSolarSystem/LaunchSites.cfg

```diff
@@ -419,6 +419,31 @@
 			}
 		}
 		Site
+                {
+                      name = nz_mahia
+                      displayName = New Zealand - Mahia
```

Comment:
The prefix should be written in ISO-3166 code, meaning that "New Zealand" should be replaced by just "NZ".

Reply:
oops... corrected now